src/rules/mv_normal_mean_covariance/covariance.jl

```diff
@@ -0,0 +1,24 @@
+
+# Variational                       # 
+# --------------------------------- #
+@rule MvNormalMeanCovariance(:Σ, Marginalisation) (q_out::Any, q_μ::Any) = begin
+    m_out, v_out   = mean_cov(q_out)
+    m_mean, v_mean = mean_cov(q_μ)
+
+    df = ndims(q_μ)
+    S  = Matrix(Hermitian(v_mean + v_out + (m_mean - m_out) * (m_mean - m_out)'))
```

Comment:
`Matrix(Hermitian(...))` is redundant for `InvWishart`

Reply:
Theoretically, yes. I wonder if that may not be the case during the actual computations.

Are you sure we can skip this step? We do the same thing for `rules/mv_norma_mean_precision/precision.jl`